packages/specs/schema/src/domain/JsonSchema.ts

```
@@ -139,27 +142,16 @@ export class JsonSchema extends Map<string, any> implements NestedGenerics {
   get class() {
     return this.getComputedType();
   }
 
   get canRef(): boolean {
-    return this.#ref;
+    return this.#isRef;
   }
 
   get isNullable(): boolean {
-    return this.#nullable;
+    return !!this.get<Boolean>("nullable");
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)